### ISSUE TITLE
Jest moduleNameMappers don't match imports with specific Webpack loaders

### ISSUE THREAD

**Issue by julianvmodesto** [NYTimes/kyt#362]
### Versions of kyt, node, OS
kyt 0.3.0
node 6.9.1
macOS 10.11.6

### Bug or Feature?
bug

### Current Behavior

In a component, I have a `import '!style-loader!css-loader!./styles.css';` to import a set of styles globally.

In the test for this component, the following error was thrown below.

```
Cannot find module '!style-loader!css-loader!./styles.css' from 'index.js'

      at Resolver.resolveModule (../node_modules/jest-resolve/build/index.js:144:17)
      ...
```
Looks like the Jest resolver didn't pick up the existing `moduleNameMapper` config for css/scss below:

https://github.com/NYTimes/kyt/blob/d803c3e633d1c46b9fc1ee353502277b26feb76f/packages/kyt-core/config/jest.js#L12

### Desired Behavior

Would be great if imports of css/scss modules with Webpack loader syntax could map to the Jest `style.stub.js`.

One fix would be to update the css/scss regex. Instead of `'^[./a-zA-Z0-9$_-]+\\.(css|scss)$'`, I modified my kyt.config.js with `\\.(css|scss)$`:

```
  modifyJestConfig: (baseConfig) => {
    const jestConfig = Object.assign({}, baseConfig);

    const styleStub = path.resolve(__dirname, 'node_modules', 'kyt', 'utils', 'jest', 'style.stub');
    jestConfig.moduleNameMapper = Object.assign({}, jestConfig.moduleNameMapper,
      {
        '\\.(css|scss)$': styleStub,
      }
    );

    return jestConfig;
  },
```

**Comment by tizmagik** [comment]
Good find. It's probably safe to change that regex, happy to review a PR with that!

**Comment by tizmagik** [comment]
Closed in #363
